model/github_app_auth.go

```diff
@@ -8,9 +8,6 @@ import (
 	"go.mongodb.org/mongo-driver/bson"
 )
 
-// kim: TODO: since this is the same data model as GitHub App Auth, should
```

Comment:
Flagging in case you still want to move this into the new directory.

Reply:
Hmm yeah I remember this one was kind of awkward because there's actually two GitHub app data models, one that stores only the app ID to cache it for internal Evergreen operations (`GitHubAppInstallation`), and one that actually stores GitHub app credentials for projects (`GitHubAppAuth`). My original intent with that was to only deal with the internal GitHub app, I didn't realize until later that the GitHub app credentials were both relying on the same shared data model.

In any case, it's the same general issue, so I can reopen DEVPROD-9272.